# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/neuwm/no-local-installdir.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Don't install in usr/local.

  - NoTag <notag@t2sde.org>

--- neuwm/meson.build.vanilla	2026-07-13 16:04:33.508077591 +0200
+++ neuwm/meson.build	2026-07-13 16:05:03.868408603 +0200
@@ -44,12 +44,12 @@
   c_args : ['-D_POSIX_C_SOURCE=200809L', '-Wall', '-Wextra'],
   dependencies : pkg_deps + [libm, luajit_dep, neuipc_dep],
   install : true,
-  install_dir : '/usr/local/bin',
+  install_dir : '/usr/bin',
 )
 
 executable('neuwmctl',
   sources : ['source/neuwmctl.c'],
   c_args : ['-D_POSIX_C_SOURCE=200809L'],
   install : true,
-  install_dir : '/usr/local/bin',
+  install_dir : '/usr/bin',
 )
